ASM/c/models.c

```diff
@@ -29,7 +30,7 @@ void load_object_file(uint32_t object_id, uint8_t *buf) {
         entry = &(z64_object_table[object_id]);
     } else {
         z64_object_table_t *extended_table = (z64_object_table_t *) (&EXTENDED_OBJECT_TABLE);
-        entry = extended_table + (object_id - num_vanilla_objects) * sizeof(z64_object_table) / 8;
+        entry = extended_table + (object_id - num_vanilla_objects - 1) * sizeof(z64_object_table_t) / 8;
```

Comment:
Just for future proofing or?

Reply:
The extended table is logically an array, so it makes more sense to use indexing than to manually calculate pointer offsets.

The extended object table size doesn't actually need to be updated since currently it has room for 4 slots with only 3 being used. I think it would still make sense to add the comment.

Reply:
Addressed in #1638.